undirected_graph.py

```diff
@@ -0,0 +1,37 @@
+from directed_graph import DirectedGraph
+
+
+class UndirectedGraph(DirectedGraph):
+
+    def add_edge(self, u, v):
```

Comment:
Make use of 'typing' library to mention this is method ovrrriding

Reply:
done